tutorial/rs/src/client.rs

```diff
@@ -39,21 +39,21 @@ fn run_client(client: &mut tutorial::CalculatorClient) {
     println!("1 + 1 = {}", client.add(1, 1).ok().unwrap());
 
     // Work: divide
-    let work = tutorial::Work {
-      op: tutorial::Operation::DIVIDE,
-      num1: 1,
-      num2: 0,
-      comment: None };
+    // let work = tutorial::Work {
```

Comment:
Could you add a FIXME with a comment explaining why this is disabled and an issue reference if there is one? Is it b/c of exceptions?